apps/src/lib/ui/ValidationStep.jsx

```diff
@@ -21,13 +21,20 @@ const style = {
   },
   header: {
     fontSize: 26,
+    lineHeight: 1.25,
```

Comment:
Another option would be to use `lineHeight: 'normal'`. According to MDN (https://developer.mozilla.org/en-US/docs/Web/CSS/line-height) this means: "Depends on the user agent. Desktop browsers (including Firefox) use a default value of roughly 1.2, depending on the element's font-family."